### PR TITLE
philmel: cleanup

### PR DESCRIPTION
**Pull request by FFHener**
I had a meeting with @patrella and we definied together how the location will look like after the maintenance work.

Please review, but do not merge as it doenst reflect the current state. I will merge it after the on site work is done

### CODE REVIEW

**Review by noxilixon — COMMENTED** [review]
:+1:

File: locations/philmel.yml [hunk 1]
```diff
@@ -9,166 +9,231 @@ community: true
...
+
+  # This section contains nearfield devices
+  - hostname: philmel-nf-o-5ghz
     role: ap
-    model: "ubnt_nanostation-m2_xm"
-    wireless_profile: freifunk_default
+    model: "mikrotik_sxtsq-5-ac"
+    mac_override: {eth0: TODO}
+
+  # TODO: For the peer GSBS2 802.11s is not working
```
[the remark this answers is not in the document]
Comment (booo) [reply locations/philmel.yml:21]: It is not that 802.11s is not working but it's only working on indoor channels or none-DFS channels with reduced output power. Therefore we need outdoor AP mode SSIDs for all towers if we deploy OpenWRT devices.

Setting up a tower with multiple nodes in 802.11s on the same channel is also a bad idea. We should follow a concept similar to the UBNT APs with the OpenWRT APs.

[the remark this answers is not in the document]
Comment (booo) [reply locations/philmel.yml:21]: As far as I remember.

The indoor channels are for indoor use and the indoor channels are quite crowded too.

File: locations/philmel.yml [hunk 2]
```diff
@@ -9,166 +9,231 @@ community: true
...
+    prefix: 10.230.2.32/28
+    ipv6_subprefix: 0
+    untagged: true
+    inbound_filtering: true
+    enforce_client_isolation: true
+    assignments:
+      philmel-core: 1
 
-  - vid: 42
+  # TODO: Change on all devices (mgmt vlan)
```
Comment (noxilixon) [inline locations/philmel.yml:208]: I am really not a fan of todo comments in a pr. Just make a issue for that.

File: locations/philmel.yml [hunk 3]
```diff
@@ -9,166 +9,231 @@ community: true
...
+    password: "file:/root/pwd"
+    daytime_limit: "2-7"
+
+# ROUTER: 10.230.2.0/24
+# --MGMT: 10.230.2.0/27
+# --MESH: 10.230.2.32/27
+# --FREE: 10.230.2.64/26
+# --DHCP: 10.230.2.128/25
+
+# TODO:give back old IP
```
Comment (noxilixon) [inline locations/philmel.yml:110]: Please resolve this, or create an extra issue and delete the todo comment.

File: locations/philmel.yml [hunk 4]
```diff
@@ -9,166 +9,231 @@ community: true
...
+  #   name: mesh_nf_w_5
+  #   prefix: 10.230.2.42/32
+  #   ipv6_subprefix: -22
+  #   mesh_ap: philmel-nf-w-5ghz
+  #   mesh_radio: 11a_standard
+  #   mesh_iface: mesh
+
+  - vid: 40
+    role: dhcp
+    prefix: 10.230.2.32/28
```
Comment (noxilixon) [inline locations/philmel.yml:200]: These ips were already used for meshing. This should match the range in comment Line 108.

**Review by noxilixon — CHANGES_REQUESTED** [review]

**Review by noxilixon — APPROVED** [review]

**Comment by Noki** [comment]
the idea with the custom wireless profile for AP meshing did not work, we should revert it and merge and simply help the guy to upgrade the router to using 802.11s.

**Pull request merged**